base-notebook/start.sh

```diff
@@ -111,6 +111,13 @@ else
     if [[ "$GRANT_SUDO" == "1" || "$GRANT_SUDO" == 'yes' ]]; then
         echo 'Container must be run as root to grant sudo permissions'
     fi
+    for f in /usr/local/bin/start-notebook.d/*; do
+      case "$f" in
+        *.sh)     echo "$0: running $f"; . "$f" ;;
+        *)        echo "$0: ignoring $f" ;;
```

Comment:
Shouldm't we run all executables in this directory, regardless of extension? Or is it explicitly required to be 'sourced' scripts?

Reply:
Right, for my use case calling `ulimit` it needs to be dot-sourced. I don't mind also running everything else if people think that's useful.